redis_gears.go

```diff
@@ -0,0 +1,159 @@
+package redis
+
+import (
+	"context"
+	"fmt"
+	"strings"
+)
+
+type gearsCmdable interface {
+	TFunctionLoad(ctx context.Context, lib string) *StatusCmd
+	TFunctionLoadArgs(ctx context.Context, lib string, options *TFunctionLoadOptions) *StatusCmd
+	TFunctionDelete(ctx context.Context, libName string) *StatusCmd
```

Comment:
Why libName and not lib?

Reply:
I used `lib` in TFunctionLoad to represent the raw library code, so I chose `libName` to represent only the name of the library.